keel-core/src/main/java/com/netflix/rocket/semver/shaded/DebianVersionComparator.java

```diff
@@ -4,7 +4,7 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-public class NetflixVersionComparator implements Comparator<String> {
+public class DebianVersionComparator implements Comparator<String> {
```

Comment:
I initially thought when @gal-yardeni renamed this that it was a good idea, but since we're just copying/shading the code here, it's easier to keep the name intact.

Reply:
So this comparator is not applicable for NPMs?

Reply:
It is, and we're still using it for NPMs, but as I said this class is copied "as is" from a Rocket code repo (hence the `shaded` in the package name), and so it's confusing to rename the class here. Any time we want to update the class (like I did in this PR), you'll have to fix the name. I thought it'd be better to keep the original name.